### DISCUSSION TITLE
Sidra tokens return

### DISCUSSION THREAD

**Discussion by naeemakhtar2001** [blockscout/blockscout#10612] [General]
How can return my Sidra tokens

**Comment by ubaidsh690o** [comment]
Someone has stolen my sidra coins.
Kindly let me know how can I recover?

**Reply by GimluCom** [reply]
This is the repo for the open-source Blockscout explorer. Please contact sidra directly about your issue.

**Reply by moduu6667** [reply]
_Thats Mean There's someone who Have your Mail and Sidra login Details Also you may know Who Done This To you Through Transaction History Hence SidraChain is Open Blockchain_

**Reply by zari1355** [reply]
با چه آدرسی تماس بگیریم؟

**Comment by Ismaildnice** [comment]
O was not verified and I mistakely contributed 200 SDA. Plz how van I get get my SDA returned.

**Reply by GimluCom** [reply]
This is the repo for the open-source Blockscout explorer. Please contact sidra directly about your issue.

**Reply by kkh55** [reply]
How to do please to have contact support sidra

**Reply by kkh55** [reply]
I have been scammed by somebody on a site how return my funds

**Reply by MuhammadRaheel0** [reply]
I mistenkly withdrawal worng address kindly fixed it plz

**Reply by MuhammadRaheel0** [reply]
@GimluCom @naeemakhtar2001 @moduu6667 @ubaidsh690o 
I mistenkly withdrawal worng address kindly fixed it plz

**Reply by GimluCom** [reply]
@MuhammadRaheel0 
This is the repo for the open-source Blockscout explorer. Please contact sidra directly about your issue.

**Comment by Ismaildnice** [comment]
I contributed 200 SDA to a project while my account was not verified. Would I benefit from the project? And if no, how can I recover my SDA. Thank you sir

Yahoo Mail: Search, organise, conquer 
 
  On Mon, 19 Aug 2024 at 22:24, ***@***.***> wrote:   


This is the repo for the open-source Blockscout explorer. Please contact sidra directly about your issue.

**Reply by GimluCom** [reply]
We have no control over that chain. Please contact sidra directly about your issue.

**Comment by kkh55** [comment]
I have been scammed by a user how to do now

**Comment by Abunaseer1** [comment]
Someone has stolen my sidra coins almost 600
on 2nd Sept 2024
What can do to get it back

**Comment by MuhammadRaheel0** [comment]
I mistenkly withdrawal worng address kindly fixed it

**Comment by davidmtunga513-dot** [comment]
hello  I have been mining sidra since 2024 and I have been never made any transactions from my sidra wallet since I started and last year someone took my 50 coins from my wallet without me knowing that and this year other one made other transactions of 729.61SDA
I'm very much anointed all this long I have been mining for someone ?
![Screenshot_20251227-071416_Samsung Internet](https://github.com/user-attachments/assets/3d708d0b-8787-40a7-b183-071f757ea318)
